cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/AddBrokerRequest.java

```diff
@@ -21,7 +21,7 @@ public AddBrokerRequest() {
   }
 
   @Override
-  protected OperationFuture handle(String uuid) {
+  public OperationFuture handle(String uuid) {
```

Comment:
why those keywords change are needed?

Reply:
This is a leftover mistake from previous iterations, I fixed it.